public/javascripts/SVLabel/src/SVLabel/modal/ModalSkip.js

```diff
@@ -164,18 +151,15 @@ function ModalSkip(form, onboardingModel, ribbonMenu, taskContainer, tracker, ui
      * Stop blinking the jump button.
      * Todo. This should be moved to LeftMenu.js
      */
-    this.stopBlinking = function() {
+    this.stopBlinking = () => {
         window.clearInterval(blinkInterval);
         uiLeftColumn.stuck.removeClass("highlight-100");
     };
 
-    // Initialize
-    uiModalSkip.unavailable.bind("click", this._handleClickUnavailable);
+    // Initialize Event Listeners
     uiModalSkip.continueNeighborhood.bind("click", this._handleClickContinueNeighborhood);
-    uiModalSkip.cancelFirst.bind("click", this._handleClickCancelFirst);
     uiModalSkip.cancelSecond.bind("click", this._handleClickCancelSecond);
```

Comment:
I think that `cancelSecond` should be renamed since there are no longer multiple different cancels to make!